model/gen-ai/spans.yaml

```diff
@@ -71,6 +71,18 @@ groups:
         requirement_level: recommended
       - ref: gen_ai.usage.output_tokens
         requirement_level: recommended
+      - ref: gen_ai.system.instructions
+        requirement_level: opt_in
+      - ref: gen_ai.input.messages
+        requirement_level: opt_in
```

Comment:
can we send skeleton (roles, etc) but no content

Reply:
alternative proposal: can we record summary

> Would it make sense to, instead of having a "skeleton", having some other attribute for reporting summary stats? (Like a count of the number of parts, and a list of the unique types contained in it)?

Reply:
I've been thinking about what'd be useful and possible to record in a summary and can't come up with good suggestions for now.

skeleton without context doesn't look useful, number of messages/content length and tool called has questionable usefullness. 

In any case, I suggest to take it as a follow up.